src/main/java/org/openpnp/machine/reference/driver/TinygDriver.java

```diff
@@ -125,6 +126,9 @@ public synchronized void connect() throws Exception {
         // Make sure we are in absolute mode
         sendCommand("G90");
         
+        // disable queue reporting
```

Comment:
Please don't check in commented out code. It looks like you implemented this above, so this should be removed.